pyproject.toml

```diff
@@ -34,18 +55,37 @@ test = [
     'scipy>=1.9,!=1.11.0,!=1.11.1',
 ]
 
-[project.entry-points."networkx.backends"]
-parallel = "nx_parallel.interface:BackendInterface"
+[tool.setuptools]
+zip-safe = false
+include-package-data = false
+packages = [
+    'nx_parallel',
+    '_nx_parallel',
+    'nx_parallel.algorithms',
+    'nx_parallel.algorithms.approximation',
+    'nx_parallel.algorithms.bipartite',
+    'nx_parallel.algorithms.centrality',
+    'nx_parallel.algorithms.connectivity',
+    'nx_parallel.algorithms.shortest_paths',
+    'nx_parallel.utils',
+]
```

Comment:
(sorry for commenting on a PR that's already merged)

It might be simpler and more maintainable to use `[tool.setuptools.packages.find]`. For example
```toml
[tool.setuptools.packages.find]
include = [
    "nx_parallel",
    "nx_parallel.*",
    "_nx_parallel",
]
```